src/clouds.cpp

```diff
@@ -191,7 +193,10 @@ void Clouds::render()
 					(float)p_in_noise_i.X * cloud_size_noise,
 					(float)p_in_noise_i.Y * cloud_size_noise,
 					m_seed, 3, 0.5);
-			grid[i] = (noise >= 0.4);
+			// normalize to 0..1 (given 3 octaves)
+			double noise_bound = 1.0 + 0.5 + 0.25;
+			double density = noise / noise_bound * 0.5 + 0.5;
+			grid[i] = (density < m_density);
```

Comment:
Since noise is now normalised, halved, added to 0.5, and clouds happen when the value is below the threshold, the default noise threshold will probably not be 0.4.